models/repo.go

```diff
@@ -29,12 +29,15 @@ import (
 
 	git "github.com/gogits/git-module"
 	api "github.com/gogits/go-gogs-client"
+	gouuid "github.com/satori/go.uuid"
 
 	"github.com/gogits/gogs/modules/bindata"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/markdown"
 	"github.com/gogits/gogs/modules/process"
 	"github.com/gogits/gogs/modules/setting"
+	"mime/multipart"
+	"io"
```

Comment:
Please adjust import ordering and grouping reference to https://github.com/Unknwon/go-code-convention/blob/master/en-US/import_packages.md